### ISSUE TITLE
PSR2.Methods.FunctionCallSignature.MultipleArguments error on short closure invoking method with multiple parameters

### ISSUE THREAD

**Issue by holtkamp** [squizlabs/PHP_CodeSniffer#2886]
Similar to https://github.com/squizlabs/PHP_CodeSniffer/issues/2572

**Versions**
codesniffer 3.5.4
PHP 7.4.3

**Description**
When adding a short syntax closure as an argument in a multiline function call which involves invoking a local method with multiple parameters there is a false positive error:

```php
class Collection
{
    private SomeOtherCollectionClass $entries;

    public function filter()
    {
        return $this->entries->filter(
            fn($entry) : bool => $this->shouldBeFiltered($entry, 'multipleParametersTriggerError');
        );
    }

    private function shouldBeFiltered($entry, string $dummyParameterToDemonstrateError) : bool
    {
        return true;
    }
}
```

```
PSR2.Methods.FunctionCallSignature.MultipleArguments: Only one argument is allowed per line in a multi-line function call
```

**Expected behavior**
No errors should be raised by the PSR-2 rule.

**Comment by gsherwood** [comment]
When I run the PSR2 standard over your sample code, all I get is:
```
FILE: temp.php
-----------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
-----------------------------------------------------------------------------------------------
 2 | ERROR | Each class must be in a namespace of at least one level (a top-level vendor name)
-----------------------------------------------------------------------------------------------
```
How are you replicating this issue?

**Comment by holtkamp** [comment]
Indeed, the example was "indicative" and flawed, my bad.

What about this:

```php
<?php

declare(strict_types=1);

namespace PSR2\Test;

class Collection
{
    private self $entries;

    public function someFilter() : self
    {
        return $this->entries->filter(
            fn($entry) : bool => $this->shouldBeFiltered($entry, 'multipleParametersTriggerError')
        );
    }

    public function filter(callable $callback) : self
    {
        return $this;
    }

    private function shouldBeFiltered($entry, string $dummyParameterToDemonstrateError) : bool
    {
        return true;
    }
}

```

Gives me:

```
FOUND 1 ERROR AFFECTING 1 LINE
--------------------------------------------------------------------------------------
 14 | ERROR | [x] Only one argument is allowed per line in a multi-line function call
--------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
```

Trying to fix using PHPCBF gives:

```
PHPCBF RESULT SUMMARY
---------------------------------------------------------------------------------------------------
FILE                                                                               FIXED  REMAINING
---------------------------------------------------------------------------------------------------
/projects/test/Collection.php     FAILED TO FIX
---------------------------------------------------------------------------------------------------
A TOTAL OF -1 ERRORS WERE FIXED IN 1 FILE
---------------------------------------------------------------------------------------------------
PHPCBF FAILED TO FIX 1 FILE
---------------------------------------------------------------------------------------------------
```

```

**Comment by gsherwood** [comment]
Sorry I missed this. I confirmed the issue in version 3.5.4 and that it was also fixed in 3.5.5.
